packages/insomnia-app/app/ui/components/dropdowns/request-actions-dropdown.js

```diff
@@ -89,6 +159,21 @@ class RequestActionsDropdown extends PureComponent {
           <DropdownHint keyBindings={hotKeyRegistry[hotKeyRefs.REQUEST_SHOW_DELETE.id]} />
         </DropdownItem>
 
+        {actionPlugins.length > 0 && <DropdownDivider>Plugins</DropdownDivider>}
+        {actionPlugins.map((p: RequestAction) => (
+          <DropdownItem
+            key={`${p.plugin.name}::${p.label}`}
+            onClick={() => this._handlePluginClick(p)}
```

Comment:
I have recently learned that if you specify the `value` prop, `DropdownItem` will feed that back as the first argument to `onClick`, so we can avoid creating an inline function here 🎉 

https://github.com/Kong/insomnia/blob/aeafe4a7c92b1d5e93f4dfe7ac9a699b0804bf45/packages/insomnia-app/app/ui/components/base/dropdown/dropdown-item.js#L19-L23

```suggestion
            value={p}
            onClick={this._handlePluginClick}
```